content/guides/authentication.md

```diff
@@ -8,17 +8,29 @@ weight: 2
 
-2. A short-lived JWT, used to do any call on the dfuse Platform, which looks like: `eyJhbGciOiJLTVNFUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1NTYxMzI4MjAsImp0aSI6IjQwNWVmOTUxLTAwZTYtNGJmNC1hZWMxLTU0NTU1ZWMzMTUwMiIsImlhdCI6MTU1NjA0NjQyMCwiaXNzIjoiZGZ1c2UuaW8iLCJzdWIiOiJ1aWQ6MHdlbnU2NmUwNzU4OWRhODY4MWNlIiwiYWtpIjoiM2NhYWEzYzA3M2FlZjVkMmYxOGUwNjJmZDkzYzg3YzMzYWIxYzA1YzEzNjI3NjU2OTgzN2Y5NDc5NzZlMjM0YSIsInRpZXIiOiJmcmVlLXYxIiwic3RibGsiOi0zNjAwLCJ2IjoxfQ.000HeTujIuS_LRvvPN6ZRCmtoZqZyV6P1enNBviwK8v7Tf7BLHJIrEpQoEREKSIMdZWPrMQl_OE55yJP0MxUDA`
+## Issuing a Long-Lived API Key
+
+Once you have created an account through the <a href="https://app.dfuse.io" title="dfuse API Portal">dfuse portal</a> you will be able to generate an API key. 
+
+Click on "GENERATE NEW KEY" and you'll now be able to start customizing your key for your specific needs.
+First assign the API key a name so that you'll be able to differentiate it from other API keys that you generate.
+Then, select the key-type between `mobile`, `web`, or `server`.
+  * `Mobile` - adaptive rate limiting to ensure abuse prevention. The rate limiting thresholds will eventually be controllable by yourself. `Mobile` can handle a large number of IPs, with more granular rate limiting by IP.
+  * `Web` - adaptive rate limiting to ensure abuse prevention. The rate limiting thresholds will eventually be controllable by yourself. JWTs generated by a `Web` key will be scoped to a small number of IPs, with heavy limiting of calls per IP. `Web` keys will have their `Origin header` verified to help with abuse prevention.
+  * `Server` - Should be used for server-to-server data access. Aggressive rate limiting for JWT issuance, as the generated tokens should be cached and used for their lifespan. Accepts a large number of calls per IP.
+  
+For ease of identification, all keys will be prefixed by their key type. `Mobile` and `web` keys are safe to publish within your mobile or web application.
 
 ## Issuing a Short-Lived JWT
 
-You use the following command to Generate a short-lived JTW from your API key. Do not forget to replace the API key by your own!
+Once you have this API key, call the [`https://auth.dfuse.io/v1/auth/issue` endpoint](#post-v1-auth-issue) to generate a fresh Authentication Token using the following command. **Do not forget to replace the API key by your own!**
 
 {{< tabs "issuing-long-jwt">}}
 {{< tab title="Request" lang="shell" >}}
-curl https://auth.dfuse.io/v1/auth/issue -s --data-binary '{"api_key":"web_abcdef12345678900000000000"}'
+curl <https://auth.dfuse.io/v1/auth/issue> -s --data-binary '{"api_key":"web_abcdef12345678900000000000"}'
```

Comment:
Why are you adding the `<` and `>`? I'm not sure why that would be necessary.

Reply:
Removed. Git blame says it was you, but I won't tell anyone ;)